opentelemetry-sdk/src/opentelemetry/sdk/metrics/export/aggregate.py

```diff
@@ -62,6 +66,11 @@ def take_checkpoint(self):
     def merge(self, other):
         with self._lock:
             self.checkpoint += other.checkpoint
+            if self.last_update_timestamp is None:
```

Comment:
I feel like when this code shows up three times, it might be good to refactor it into a common utility method. All the timestamp comparisons seem to be doing the same logic.

Reply:
I agree with taking out repetitive code. Would passing an aggregator into a utility method look weird? The observer aggregator also modifies `last`, so it is not _exactly_ the same; type checking would be needed. Is that logic worth it to simply reduce some duplicate code?

Reply:
I was considered passing in two Optional[DateTime] objects, and doing the none check there
you could pass the object in itself, but I agree that feels weird

Reply:
I made a change that factors out the checking logic and assigns the values as well.